src/components/Dashboard/TodoList.vue

```diff
@@ -1172,6 +1175,54 @@ export default class TodoList extends Mixins(
     }
   }
 
+  private async loadChangeOfRegistration (task: ApiTaskIF): Promise<void> {
+    const filing = task.task.filing
+    const header = filing.header
+    const changeOfRegistration = filing.changeOfRegistration
+
+    // NB: don't check "changeOfRegistration" as it may be empty
+    if (header) {
+      const title = `Change to ${this.getCorpTypeDescription(this.getEntityType)} Registration`
+
+      // set subtitle only if DRAFT
+      let subtitle: string = null
+      if (this.isStatusDraft(header)) {
+        if (this.nameRequest) {
+          subtitle = `NR APPROVED - ${this.expiresText(this.nameRequest)}`
+        } else {
+          subtitle = 'DRAFT'
+        }
+      }
+
+      const paymentStatusCode = header.paymentStatusCode || null
+      const payErrorObj = paymentStatusCode && await this.getPayErrorObj(paymentStatusCode)
```

Comment:
Does `getPayErrorObj()` never throw an error (that needs to be handled here)? (It might catch its own errors and always return something sane, I dunno.)

Reply:
It's an interesting request. 
It will make a request to fetch any errors associated with the code, return them if there is errors. 
Catches itself is there is an error and ignores them. 
So on this level it would be payErrorObj  would be `false`

Reply:
Perfect. Thanks for confirming!